python/ngen_cal/src/ngen/cal/parameter.py

```diff
@@ -11,5 +11,13 @@ class Parameter(BaseModel, allow_population_by_field_name = True):
     min: float
     max: float
     init: float
+    alias: Optional[str]
+
+    @root_validator
```

Comment:
Minor improvement that without could result in a misleading error.

```suggestion
    @root_validator(skip_on_failure=True)
```